code/API_definitions/BYON-Webrtc-Events.yaml

```diff
@@ -0,0 +1,1520 @@
+
+        **NOTE**: The `Unregistered` status will be only provided by the notification service, a GET /racm/sessions/ for an unregistered endpoint will produce an 404 response.
+      type: object
+      properties:
+        subscriptionId:
+          $ref: "#/components/schemas/SubscriptionId"
+        msisdn:
+          type: string
+          description: The MSISDN is the mapping of the telephone number to the subscriber identity module in a mobile or cellular phone system. A unique identifier on the network to this subscriber.
+        regStatus:
+          $ref: '#/components/schemas/RegistrationStatus'
+        sessionRequest:
+          description: Object present only when receiving a new session request
+          type: object
+          properties:
+            originator:
+              $ref: "#/components/schemas/SubscriberUser"
+            receiver:
+              $ref: "#/components/schemas/SubscriberUser"
+            subject:
+              type: string
+              description: Subject of the call
+            sessionId:
+              type: string
+              description: Unique identifier of the session to subscribe to notifications
+
+    RegistrationStatus:
+      type: string
+      enum:
+        - Registered
+        - Unregistered
+
+    EventSession:
+      description: event structure for session events
+      allOf:
+        - $ref: "#/components/schemas/CloudEvent"
+        - type: object
+          properties:
+            data:
+              $ref: "#/components/schemas/SessionInformation"
+
+    SessionInformation:
+      description: |-
+        A `org.camaraproject.webrtc-events.registration` event includes the same session information received using the WebRTC BOYN CallHandling API, when retrieving information from the GET /vvoip/session/{vvoipSessionID} endpoint.
+
+        In general lines, it is receive each time that an event happens with the session. This might happen when a call is received, when SDP updates are included for any party or when the call is finished due an ordered cleanup or a service outage.
+
+        Regarding the SDP descriptions for offer and answer:
+        - The offer, which MUST be present in a request from the application to the server to create a session. Note that the offer can be absent in a session created by the server as part of an offerless INVITE [RFC3261].
+        - For the answer. This type represents an answer in WebRTC Signaling. This element is not present in case there is no answer yet, or the session invitation has been declined by the Terminating Participant. This element MUST NOT be present in a request from the application to the server to create a session.
+      required:
+        - vvoipSessionID
+      type: object
+      properties:
```

Comment:
Is `subscriptionId` missing here?

Reply:
`subscriptionId` included as per comms 4.0 -> https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#122-event-notification

Reply:
Fixed on commit [248e056](https://github.com/camaraproject/WebRTC/pull/53/commits/248e056ce8ef874809066188fe56629f271f774f)

Resolving